layouts/partials/cards/_header.html

```diff
@@ -0,0 +1,18 @@
+<div class="flex items-center {{ if isset . "inlineLink" }}transform-gpu translate-x-0 group-hover:translate-x-2 transition-all{{ end }}">
+  {{ if isset . "icon"}}
+  <div class="mr-8">
+    <img src="{{ .icon }}" />
+  </div>
+  {{ end }}
+  <div>
+    <div class="font-bold text-white text-lg uppercase tracking-widest">
+      {{ .name }}
+    </div>
+      {{ if isset . "patch"}}
```

Comment:
for a lot of these if guards, you can probably replace that with a [with](https://gohugo.io/functions/with/) statement, since it will omit the entire block if it isn't found.